sklearn/pipeline.py

```diff
@@ -1232,6 +1232,11 @@ def n_features_in_(self):
         # X is passed to all transformers so we just delegate to the first one
         return self.transformer_list[0][1].n_features_in_
 
+    def __sklearn_is_fitted__(self):
+        # Delegate whether feature union was fitted
+        check_is_fitted(self.transformer_list[0][1])
```

Comment:
I think we need to check all the transformers to make sure the whole `FeatureUnion` is fitted.

Also `transformer_list` can be configured with "passthrough" and "drop" as the estimator, so those needs to be avoided in this check. (In other words, `transformer_list[0][1]` can be string.)

Reply:
Thanks @thomasjpfan. Let me know if my fix works. Tx